Dockerfile-context

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Remove this file

Reply:
We need it :). We are using it to get list of all the files that are included in the Docker context (to change permissions). I will add some more comments explaining the purpose of it.

Reply:
Could we use `git ls-files` or similar instead?

Reply:
I already tried it before. We can't use it because .dockerignore is quite different from .gitignore. Also in our case we use the different (recommended) pattern for dockerignore: exclude all + include only what's needed. This is much better for docker context stability. And simply having a separate simple docker like that one is a super easy way to see which files are added to context (then we can change the +g/-g flags for only those files - for docker cache invalidation purpose).